src/Symfony/Component/Console/Tests/ApplicationTest.php

```diff
@@ -403,13 +404,14 @@ public function testFindCaseInsensitiveAsFallback()
 
     public function testFindCaseInsensitiveSuggestions()
     {
-        $this->expectException(CommandNotFoundException::class);
-        $this->expectExceptionMessage('Command "FoO:BaR" is ambiguous');
         $application = new Application();
         $application->add(new \FooSameCaseLowercaseCommand());
         $application->add(new \FooSameCaseUppercaseCommand());
 
-        $this->assertInstanceOf(\FooSameCaseLowercaseCommand::class, $application->find('FoO:BaR'), '->find() will find two suggestions with case insensitivity');
```

Comment:
an exception is expected, I think this assertion is not needed, isn't it? 

cc @stof

Reply:
the assertion is indeed dead code.